### PR TITLE
Revise RPM upgrade

### LINKED ISSUE #1458: Improve RPM upgrade documentation
### Description

Address issue with instructions for RPM upgrade

### Discussion

_No response_

### Motivation

_No response_

### Details

_No response_

### PR DESCRIPTION
**Pull request by andymunro**
Applying input from support

Closes #1458

@pskopek, @pdrozd Can you verify this is accurate?

### CODE REVIEW

**Comment by andymunro** [comment]
@pdrozd This is needed for 7.5.2 on Tuesday

**Comment by nehachopra27** [comment]
@andymunro: JFYI, I am picking this up.

**Comment by nehachopra27** [comment]
Hi @andymunro ,
Please backport the changes in the product MR to this upstream PR.
Thanks

**Comment by andymunro** [comment]
thanks, @nehachopra27 This is backported here: https://gitlab.cee.redhat.com/keycloak/keycloak-documentation-prod/-/merge_requests/84

**Comment by andymunro** [comment]
@nehachopra27 Please let me know if there are further changes needed.

**Comment by nehachopra27** [comment]
@andymunro: Thanks Andy. It is ready to be merged.

**Review by pdrozd — APPROVED** [review]

File: upgrading/topics/install_new_version.adoc [hunk 1]
```diff
@@ -65,22 +60,43 @@ For Red Hat Enterprise Linux 8:
...
+ repo id repo name
+ rh-sso-7.5-for-rhel-8-x86_64-rpms Single Sign-On 7.5 for RHEL 8 x86_64 (RPMs)
+ rhel-8-for-x86_64-appstream-rpms Red Hat Enterprise Linux 8 for x86_64 - AppStream (RPMs)
+ rhel-8-for-x86_64-baseos-rpms Red Hat Enterprise Linux 8 for x86_64 - BaseOS (RPMs)
+
+. Back up any modified configuration files and custom modules.
+
+. Remove packages for RH-SSO 7.4.
++
+Normally the old packages are removed by `dnf upgrade`. However, the RH-SSO 7.5 packages use a format that conflicts with files handled by the JBoss EAP 7.3 package. Therefore, the RH-SSO 7.4 and JBoss EAP 7.3 packages must be removed.
```
Comment (iankko) [inline upgrading/topics/install_new_version.adoc:83]: @andymunro Maybe add `manually` at the end of the last sentence? IOW, it to read like `Therefore, the RH-SSO 7.4 and JBoss EAP 7.3 packages must be removed manually.`

**Review by iankko — APPROVED** [review]
LGTM Thanks, Andrew!

**Review by abstractj — APPROVED** [review]

**Comment by andymunro** [comment]
Thanks, @abstractj

**Pull request merged**